reports/California.md

```diff
@@ -805,6 +805,20 @@ id: ca-losangeles-44
 * https://www.reddit.com/r/2020PoliceBrutality/comments/hzol2y/lapd_standoff_with_protesters_during_a_portland/
 
 
+### Police fire less lethals at bystanders and peaceful protesters | July 25th
+
+A woman who was recording protests was shot with less lethals by LAPD officers. She states that bystanders and peaceful protesters were struck.
+
+tags: shoot, projectile, protester, bystander
+
```

Comment:
I would maybe remove the bystander tag since it isn't super clear that bystanders were caught up in